src/Neo/SmartContract/StorageKey.cs

```diff
@@ -79,5 +80,17 @@ public byte[] ToArray()
             }
             return cache;
         }
+
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public static implicit operator StorageKey(byte[] value)
+        {
+            return new StorageKey(value);
+        }
+
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public static implicit operator StorageKey(ReadOnlyMemory<byte> value)
+        {
+            return value.ToArray();
```

Comment:
```suggestion
            return value.Span;
```

`Span` is faster and less memory. I don't think you need to add `Span` to `ctor`, but we should be using `Span<>`, `ReadOnlySpan<>` for non-threaded and `ReadOnlyMemory<>` and `Memory<>` for `async`